velox/core/PlanNode.h

```diff
@@ -1232,6 +1232,8 @@ class HashJoinNode : public AbstractJoinNode {
       VELOX_USER_CHECK(
           isNullAwareSupported(joinType),
           "Null-aware flag is supported only for semi and anti joins");
+      VELOX_USER_CHECK_EQ(
```

Comment:
Could we add a failure unit test in HashJoinTest that expect a failure if there are more than one join keys in null aware join types?

Reply:
Sure. Added a test.